paymentsheet/res/layout/stripe_google_pay_button.xml

```diff
@@ -2,48 +2,43 @@
 <merge xmlns:android="http://schemas.android.com/apk/res/android">
 
     <RelativeLayout
+        android:id="@+id/google_pay_button_layout"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/stripe_paymentsheet_max_primary_button_height"
-        android:orientation="vertical">
-        
+        android:layout_height="@dimen/stripe_paymentsheet_googlepay_button_height"
+        android:paddingTop="2sp"
+        android:background="@drawable/googlepay_button_background"
```

Comment:
The background looks quite a bit fuzzy, especially around the corners. Is there anything we can do here?

Reply:
Where does it looks fuzzy? It looks good for me, it's using the 9-patch that are provided for all density levels.

Reply:
I find that the corner looks fuzzy, as compared to the Link button. I’m not sure what can be done to avoid that though, if we’re already using the 9-patch images with the correct densities 🤷‍♂️

![Screenshot_20220901_123215](https://user-images.githubusercontent.com/110940675/187966242-6ba35e8c-741a-4604-9829-2e255c758cbb.png)

Reply:
Oh well, guess we’re forced to use Google’s assets 🤷‍♂️